pkg/scheduler/scheduler.go

```diff
@@ -275,9 +260,8 @@ func New(client clientset.Interface,
 		podInitialBackoffSeconds: options.podInitialBackoffSeconds,
 		podMaxBackoffSeconds:     options.podMaxBackoffSeconds,
 		enableNonPreempting:      utilfeature.DefaultFeatureGate.Enabled(kubefeatures.NonPreemptingPriority),
+		profiles:                 append([]schedulerapi.KubeSchedulerProfile(nil), options.profiles...),
```

Comment:
Any difference with:

```go
profiles: options.profiles,
```
?

Reply:
Yes. This was actually a source of an initial bug I had. The configurator overrides `profiles` with the processed configuration coming from the provider or policy. If we don't do a copy, the `defaultSchedulerOptions` is overwritten.